FinderKeeper/Assets/Scripts/Collect.cs

```diff
@@ -28,14 +31,14 @@ public void HideCollectionPanel()
         }
     }
 
-    void OnCollisionEnter(Collision collision)
+  /*  void OnCollisionEnter(Collision collision)
```

Comment:
Is this still needed? Either uncomment or delete it.

Reply:
Deleted